routes/api.py

```
@@ -0,0 +1,512 @@
+            for rule in resource.spec.rules or []:
+                for path in rule.http.paths:
+                    if path.backend.service:
+                        service_name = path.backend.service.name
+                        try:
+                            service = v1.read_namespaced_service(namespace, service_name)
+                            related_resources['Services'].append({
+                                'name': service.metadata.name,
+                                'status': 'Active',
+                                'info': f'Targeted via {rule.host}{path.path}'
+                            })
+                        except:
```

Ruff (0.8.0)

Do not use bare `except`

(E722)